### ISSUE TITLE
Ideally, we want to be able to filter for X … and things 1 connection away

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-2018#295]
For example, if we filter for people, the network is completely empty. So we'd want to filter for people, and have an option of also seeing things a distance of 1 away from them. The example that came up was being able to see what different things boys and girls were interested in (ignoring the gender binary issue for the moment).

**Comment by benloh** [comment]
This one seems difficult, mostly because it breaks the Filter UI. I guess we would add a checkbox "Show nodes one degree away" or "Show connected nodes (one degree)" or something like that?

@jdanish Sorry I missed this because it wasn't assigned to an Iteration.  Did you want this for Spring (now) or Fall (later)?

**Comment by jdanish** [comment]
Yeah - that’s what I was thinking. Definitely later - this was one of the “so Sri knows for pondering architecture” issues for now.----(from my iPhone)Joshua Danishhttp://www.joshuadanish.comOn Jun 28, 2023, at 10:29 AM, benloh ***@***.***> wrote:﻿
This one seems difficult, mostly because it breaks the Filter UI. I guess we would add a checkbox "Show nodes one degree away" or "Show connected nodes (one degree)" or something like that?
@jdanish Sorry I missed this because it wasn't assigned to an Iteration.  Did you want this for Spring (now) or Fall (later)?
